lib/resize.js

```diff
@@ -2,9 +2,21 @@ const Jimp = require('jimp');
 const TinyColor = require('tinycolor2');
 
 module.exports = async (options, buffer) => {
```

Comment:
might be cleaner to do `({ background, type, width, height }, buffer)`. thoughts?